### PR TITLE
Fix for DNS widget value output

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

**Review by xpositivityx — APPROVED** [review]

File: massdriver-application-azure-app-service/dns.tf [hunk 1]
```diff
@@ -1,6 +1,10 @@
+locals {
+  zone_name = element(split("/", var.dns.zone_id), length(var.dns.zone_id) - 2)
```
Comment (WillBeebe) [inline massdriver-application-azure-app-service/dns.tf:2]: This is close but `length` of the string `var.dns.zone_id` won't get us the right value here. I think you wanted length of the split array.

Comment (WillBeebe) [inline massdriver-application-azure-app-service/dns.tf:2]: When I counted before, I believe 8 is the right index but def check my work.

```suggestion
  zone_name = element(split("/", var.dns.zone_id), 8)
```

Comment (mclacore) [reply massdriver-application-azure-app-service/dns.tf:2]: This is what I get when testing in console: 
image

Comment (xpositivityx) [reply massdriver-application-azure-app-service/dns.tf]: I might store the split as a local and reference that in both places. We cant be sure that magic number will always work out.

Comment (WillBeebe) [reply massdriver-application-azure-app-service/dns.tf]: Yes, but I think this is what's going on. 

```terraform
# you're getting the length of the string
length("/one/two/three/four")
19

# when the index passed to element is greater than the number of elements in the array
# it returns the last one
element(["one", "two", "three"], 17)
"three"
```

**Review by WillBeebe — APPROVED** [review]

**Pull request merged**